bump.json

```diff
@@ -0,0 +1,24 @@
+{
+    "release": {
+        "changelog": {
+            "file": "CHANGELOG.md",
+            "create": true,
+            "open": true 
+        },
+        "commit": {
+            "enabled": true,
+            "message": "Bumped version to [version]"
+        },
+        "tag": {
+            "enabled": true,
+            "name": "v[version]"
+        },
+        "github": {
+            "enabled": true,
+            "open": true,
+            "draft": true,
+            "prerelease": true,
+            "token": ""
```

Comment:
A ``GIT_HUB`` token should be updated here in order to push the newly tagged version and drat a release :)